app/assets/stylesheets/global.scss

```diff
@@ -25,6 +63,35 @@
   margin-bottom: -7px;
 }
 
+h1 {
+  font-family: "assassin";
+}
+
+.title-bar {
+  height: 7em;
+}
+
+a {
+  font-family: "Typography_Times";
+  font-style: bold;
+}
+
+#flexbox {
+  -webkit-box-orient: horizontal;
+  display: -webkit-box;
+  height: 100%;
+}
+
+#registration {
```

Comment:
Address this either by updating the styling or your .yml file (and make a comment here saying you did so in this PR)